oak-run/src/test/java/org/apache/jackrabbit/oak/run/DataStoreCommandTest.java

```diff
@@ -390,6 +423,95 @@ public void testConsistencyVerboseRootPathWithoutVerbose() throws Exception {
         assertFalse(true);
     }
 
+    @Test
+    public void testDumpRefWithVerboseRootPath() throws Exception {
+        File dump = temporaryFolder.newFolder();
+        Data data = prepareData(storeFixture, blobFixture, 10, 4, 1);
+        storeFixture.close();
+        additionalParams += " --verboseRootPath /c1,/c2";
+
+
+        for (String id : data.idToPath.keySet()) {
+            if (data.idToPath.get(id).equals("/c1") || data.idToPath.get(id).equals("/c2")) {
+                data.addedSubset.add(id);
+            }
+        }
+        testDumpRef(dump, data, true, true);
+    }
+
+    @Test
+    public void testDumpRef() throws Exception {
+        File dump = temporaryFolder.newFolder();
+        Data data = prepareData(storeFixture, blobFixture, 10, 4, 1);
+        storeFixture.close();
+
+        testDumpRef(dump, data, true, false);
+    }
+
+    @Test
+    public void testDumpRefWithoutVerbose() throws Exception {
+        File dump = temporaryFolder.newFolder();
+        Data data = prepareData(storeFixture, blobFixture, 10, 4, 1);
+        storeFixture.close();
+
+        testDumpRef(dump, data, false, false);
+    }
+
+    @Test
+    public void testDumpRefVerboseRootPathRegex() throws Exception {
+        File dump = temporaryFolder.newFolder();
+        Data data = prepareData(storeFixture, blobFixture, 10, 4, 1, true);
+        storeFixture.close();
+
+        additionalParams += " --verboseRootPath /foo --verbosePathInclusionRegex /*/test";
+
+        for (String id : data.idToPath.keySet()) {
+            if (data.idToPath.get(id).contains("/foo")) {
+                data.addedSubset.add(id);
+            }
+        }
+
+        testDumpRef(dump, data, true, true);
+    }
+
+    @Test
+    public void testDumpRefVerboseRegexWithoutVerboseRootPath() throws Exception {
+        File dump = temporaryFolder.newFolder();
+        Data data = prepareData(storeFixture, blobFixture, 10, 4, 1);
+        storeFixture.close();
+
+        additionalParams += " --verbosePathInclusionRegex /*/*";
+
+        try {
+            // Create exec command without --verboseRootPath - It should throw an exception
+            // since we used --verbosePathInclusionRegex without --verboseRootPath
+            testDumpRef(dump, data, true, false);
+        } catch (OptionException e) {
+            assertTrue(e.getMessage().equals("Option(s) [verbosePathInclusionRegex] are unavailable " +
+                    "given other options on the command line"));
+            return;
+        }
+        assertFalse(true);
+    }
+
+    @Test
+    public void testDumpId() throws Exception {
+        File dump = temporaryFolder.newFolder();
+        Data data = prepareData(storeFixture, blobFixture, 10, 4, 1);
+        storeFixture.close();
+
+        testDumpIds(dump, data, true);
+    }
+
+    @Test
+    public void testDumpIdWithoutVerbose() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5079914)